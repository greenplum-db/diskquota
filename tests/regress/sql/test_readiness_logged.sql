CREATE DATABASE test_log_readiness;
\c test_log_readiness

CREATE TABLE t (i int) DISTRIBUTED BY (i);

CREATE EXTENSION diskquota;
SELECT pg_sleep(1); --Wait for the check completes

SELECT count(*) FROM gp_toolkit.gp_log_database
WHERE logmessage = '[diskquota] diskquota is not ready';

\! gpstop -raf -> /dev/null
\c
SELECT pg_sleep(1); --Wait for the check completes

SELECT count(*) FROM gp_toolkit.gp_log_database
WHERE logmessage = '[diskquota] diskquota is not ready';

DROP EXTENSION diskquota;

\c contrib_regression
DROP DATABASE test_log_readiness;
