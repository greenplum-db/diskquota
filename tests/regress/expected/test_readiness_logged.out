CREATE DATABASE test_readiness_logged;
\c test_readiness_logged
CREATE TABLE t (i int) DISTRIBUTED BY (i);
CREATE EXTENSION diskquota;
WARNING:  [diskquota] diskquota is not ready because current database is not empty
HINT:  please run 'SELECT diskquota.init_table_size_table();' to initialize diskquota
SELECT diskquota_test.wait('SELECT diskquota_test.check_cur_db_status(''UNREADY'');');
 wait 
------
 t
(1 row)

SELECT count(*) FROM gp_toolkit.gp_log_database
WHERE logmessage = '[diskquota] diskquota is not ready';
 count 
-------
     1
(1 row)

\! gpstop -raf > /dev/null
\c
SELECT diskquota_test.wait('SELECT diskquota_test.check_cur_db_status(''UNREADY'');');
 wait 
------
 t
(1 row)

SELECT count(*) FROM gp_toolkit.gp_log_database
WHERE logmessage = '[diskquota] diskquota is not ready';
 count 
-------
     2
(1 row)

DROP EXTENSION diskquota;
\c contrib_regression
DROP DATABASE test_readiness_logged;
